dosdp/README.md

```diff
@@ -0,0 +1,13 @@
+# Dead simple owl design pattern (DOSDP) exchange format
+
+Homepages
+* https://github.com/INCATools/dead_simple_owl_design_patterns -- DOSDP specification, context and docs
+* https://github.com/INCATools/dosdp-tools -- DOSDP tools - a command-line utility for working with DOSDP design patterns and OWL ontologies.
+
+Docs
+* https://github.com/INCATools/dead_simple_owl_design_patterns/blob/master/spec/DOSDP_schema_full.yaml (Full schema)
+* http://incatools.github.io/dead_simple_owl_design_patterns/
+
+Contact
+* David Osumi-Sutherland cjmungall AT lbl DOT gov @dosumis
+* Jim Balhoff balhoff AT renci DOT org @balhoff
```

Comment:
@matentzn Did you want yourself here too?

Reply:
Thank you @davidlehn, both @dosumis and @balhoff are avid GitHub users and I would like DOSDP related communications to go through them. Thank you!

Reply:
Ok.  Can one of you approve this?  Also is that first email correct or copy & pasted? (We don't even require emails here.  Github ids are usually enough.  It's mostly for tagging people and to check access control.)

Reply:
The reason for asking for approval is that we want at least one maintainer.  I see some of the requested rules and doc links don't work already.  Good to have a real maintainer listed that can deal with such things.

Reply:
OMG this was embarassing. No PRs at 11 PM no more.. :D Sorry. Ok I added myself now.